source/Generic/PlayState/Views/PlayStateManagerView.xaml

```diff
@@ -39,26 +40,34 @@
                           ItemsSource="{Binding PlayStateDataCollection}">
                         <ListView.View>
                             <GridView>
-                                <GridViewColumn Width="250" Header="{DynamicResource LOCPlayState_ManagerGameLabel}">
+                                <GridViewColumn Width="42">
                                     <GridViewColumn.CellTemplate>
                                         <DataTemplate>
-                                            <TextBlock Text="{Binding Game.Name}" VerticalAlignment="Center"/>
+                                            <Image Height="32" Width="32" HorizontalAlignment="Center" VerticalAlignment="Center" 
+                                                   Source="{Binding Game.Icon, Converter={StaticResource IconPathConverter}}" />
```

Comment:
Regarding the icons loading, can you check if there are not issues deleting or replacing the icon file when it has been loaded by the window? Since it's loading the image it's probable that it will lock the image file

Reply:
If you use Playnite to replace or delete it, it will work as expected. If you use windows file explorer, it will happen as you say, but I think this shouldn't be the proper way to replace / delete the image file.

Reply:
I don't like that the file is being locked because we don't know if Playnite itself, an extension or even if the user will try to modify the image at any moment. To prevent this issue you can use the Playnite `ImageStringToImageConverter` converter, which loads the image file without locking it. Here's an example: https://github.com/darklinkpower/PlayniteExtensionsCollection/blob/a0285327cf7f312d51871d60423c13b3a4ccda6d/source/Library/JastUsaLibrary/Views/GameDownloadsView.xaml#L20-L22